subsys/mgmt/hawkbit/Kconfig

```diff
@@ -145,6 +145,11 @@ config HAWKBIT_STATIC_CERT_TAG
 
 endif
 
+config HAWKBIT_REBOOT_COLD
+	bool "Cold reboot after update"
+	help
+	  Do a cold reboot instead of a warm reboot.
```

Comment:
IMO the "instead...." in the help message really makes it look that it would be more suited as a choice -- `HAWKBIT_REBOOT_MODE` or something, that's either `WARM` or `COLD` and defaults to WARM?
thoughts?